client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -209,6 +209,13 @@ class PurchaseMeta extends Component {
 				return translate( 'Credits' );
 			}
 
+			if (
+				( isExpired( purchase ) || isExpiring( purchase ) ) &&
+				! isPaidWithCredits( purchase )
+			) {
+				return '—';
```

Comment:
While I'm sure this `-` is fine in most cases, I see that below there's a `return translate( 'None' );` so maybe we should re-use that string instead? Or did you intentionally want to differentiate this value?

Reply:
I wanted it to show that nothing was associated and `None` would work just as well. I'll be happy to update it.